Dockerfile

```diff
@@ -1,27 +1,29 @@
-FROM node:lts-alpine3.11
+FROM node:lts-alpine3.11 AS base
 
 # Versions
 # ARG NODE_VERSION=12.14.0 # Alpine 3.11 tracks NodeJS 12.x LTS
 # ARG NPM_VERSION=6.13.4 # Alpine 3.11 tracks NPM 6.x
-ARG SAILS_VERISON=^1.0.0
-
-# Variables
-ENV WORKDIR "/cppd-medical-report-sails"
 
-# Copy in files
-WORKDIR ${WORKDIR}
+ARG SAILS_VERISON=^1.0.0
+WORKDIR /app
 
-# Install SailsJS
-RUN npm install sails@${SAILS_VERISON} -g
+# Need to install python to compile bcrypt dependency
+RUN apk --no-cache add --virtual native-deps \
+  g++ gcc libgcc libstdc++ linux-headers autoconf automake make nasm python git && \
+  npm install --quiet node-gyp -g
 
-# Copy in build files
-COPY package.json package-lock.json ./
+# Package files
+COPY . /app/
```

Comment:
Why did you remove copying over the package.json files? This will increase the build time significantly.

Reply:
... good question... I just pushed a change, see how it works.